src/toil/test/wdl/wdltoil_test.py

```diff
@@ -90,6 +114,17 @@ def check(self, p: subprocess.CompletedProcess) -> None:
 
         p.check_returncode()
 
+    @slow
+    def test_unit_tests_v11(self):
+        # There are still some bugs with the WDL spec, use a fixed version until
+        repo_url = "https://github.com/stxue1/wdl.git"
```

Comment:
Hmmm... has this been made into a PR for `openwdl`?

Reply:
Opened one at https://github.com/openwdl/wdl/pull/689